channelserver/src/meta.rs

```diff
@@ -127,14 +127,10 @@ fn get_ua(
     None
 }
 
-fn is_trusted_proxy(proxy_list: &[IpNet], host: &str) -> Result<bool, HandlerError> {
+fn is_trusted_proxy(proxy_list: &[IpNet], host: &IpAddr) -> Result<bool, HandlerError> {
     // Return if an address is NOT part of the allow list
```

Comment:
did I read this comment wrong or is it describing the opposite of what's happening?

Reply:
Sigh, no, you read it right. It's wrong. Thanks!